spec.go

```diff
@@ -437,6 +437,9 @@ func createCgroupConfig(name string, spec *specs.LinuxRuntimeSpec, devices []*co
 	if err != nil {
 		return nil, err
 	}
+	if spec.Linux.CgroupsPath != "" {
```

Comment:
We use a combination of parent and the name to derive the path. So, you need to handle the parent as well and make sure it is correct.